### ISSUE TITLE
Any chance to see microsoft teams officially supported?

### ISSUE THREAD

**Issue by TeoColuccio** [Fmstrat/winapps#130]
Although Microsoft Teams is available for Linux, this is in preview version and lacks some (actually many) features found in the Windows version.  It would therefore be really interesting to be able to use this full version, even if I realize that it is not easy to manage the video stream of a videoconference in a virtual machine and there could be lag problems.  I hope it is possible.

**Comment by Tiebe** [comment]
Any program can be added... RTFM

**Comment by TeoColuccio** [comment]
yes, in the readme it says that every application works .. but my question was: "will we ever see Microsoft teams in officially supported applications?".  It is one thing to have the ability to run an app and quite another to have some certainty that it works, i.e. if it becomes part of the supported applications.

**Comment by Tiebe** [comment]
If you want I can do this for you and I'll make a pull request

**Comment by TeoColuccio** [comment]
> If you want I can do this for you and I'll make a pull request

would be great

**Comment by Tiebe** [comment]
I've made it, check Pull Request #142

**Comment by TeoColuccio** [comment]
> I've made it, check Pull Request #142

thank you very much

**Comment by Tiebe** [comment]
No problem!

**Comment by LyzardKing** [comment]
Does this work? I'm trying to make Skype For Business work, since it's not available on Linux at all..
Unfortunately I can't get the microphone to work (and if I set audio-mode:1 I lose the output, so no audio at all)
